src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img
+        class="logo"
+        src="./images/logo.png"
+        alt="logo-moyo"
+      />
```

Comment:
Need to use the tag a here
```suggestion
      <a>
       <img
         class="logo"
         src="./images/logo.png"
         alt="Moyo logo"
       />
     </a>
```